src/directives/scaleselector.js

```diff
@@ -243,6 +247,7 @@ ngeo.ScaleselectorController.prototype.handleResolutionChange_ = function(e) {
  */
 ngeo.ScaleselectorController.prototype.handleViewChange_ = function(e) {
   this.registerResolutionChangeListener_();
+  this.handleResolutionChange_(null);
```

Comment:
Yes it's still useful.
The ``change:view`` event is only fired when a new view is associated to the map. 
The ``change:view`` event is not fired when something into the view changes (i.e. center, resolution, ...)

Then when a new view is associated to the map, first we have to unregister the event ``change:resolution`` associated  to the old view object and then register a new listener on event ``change:resolution``  to the new view. 

Is it now more understandable ?